embrace-android-sdk/src/test/java/io/embrace/android/embracesdk/internal/logs/EmbraceLogServiceTest.kt

```diff
@@ -0,0 +1,119 @@
+package io.embrace.android.embracesdk.internal.logs
+
+import com.google.common.util.concurrent.MoreExecutors
+import io.embrace.android.embracesdk.LogExceptionType
+import io.embrace.android.embracesdk.Severity
+import io.embrace.android.embracesdk.fakes.FakeMetadataService
+import io.embrace.android.embracesdk.fakes.FakeOpenTelemetryLogger
+import io.embrace.android.embracesdk.fakes.FakeSessionIdTracker
+import io.embrace.android.embracesdk.internal.utils.Uuid
+import io.embrace.android.embracesdk.worker.BackgroundWorker
+import io.mockk.every
+import io.mockk.mockkStatic
+import org.junit.Assert.assertEquals
+import org.junit.Assert.assertNotEquals
+import org.junit.Assert.assertNull
+import org.junit.Before
+import org.junit.BeforeClass
+import org.junit.Test
+
+internal class EmbraceLogServiceTest {
+
+    companion object {
+        private lateinit var logger: FakeOpenTelemetryLogger
+        private lateinit var metadataService: FakeMetadataService
+        private lateinit var sessionIdTracker: FakeSessionIdTracker
+
+        @BeforeClass
+        @JvmStatic
+        fun beforeClass() {
+            metadataService = FakeMetadataService()
+            sessionIdTracker = FakeSessionIdTracker()
+            mockkStatic(Uuid::class)
+            every { Uuid.getEmbUuid() } returns "id"
```

Comment:
What are these mocks required for? Is there any way we could replace this with fakes?

Reply:
Removed the mock